kfs-core/src/main/java/edu/arizona/kfs/gl/service/impl/ScrubberValidatorImpl.java

```diff
@@ -219,16 +220,15 @@ protected Message validateGTEAccount(OriginEntryFull originEntry) {
         account.refreshReferenceObject("subFundGroup");
 
         if(ObjectUtils.isNull(account)){
-            throw new IllegalArgumentException("This account specified "+originEntry.getChartOfAccountsCode()+"-"+originEntry.getAccountNumber()+" does not exist. For sequence "+originEntry.getTransactionLedgerEntrySequenceNumber());
+            return MessageBuilder.buildMessageWithPlaceHolder(edu.arizona.kfs.sys.KFSKeyConstants.ERROR_GLOBAL_TRANSACTION_EDIT_SCRUBBER_INVALID_VALUES, Message.TYPE_FATAL, new Object[] {"Account", originEntry.getChartOfAccountsCode() + "-" + originEntry.getAccountNumber()});
         }
         if(ObjectUtils.isNull(originEntry.getFinancialObject())){
-            throw new IllegalArgumentException("This account specified "+originEntry.getFinancialObjectCode()+"-"+originEntry.getAccountNumber()+" does not exist. For sequence "+originEntry.getTransactionLedgerEntrySequenceNumber());
+            return MessageBuilder.buildMessageWithPlaceHolder(edu.arizona.kfs.sys.KFSKeyConstants.ERROR_GLOBAL_TRANSACTION_EDIT_SCRUBBER_INVALID_VALUES, Message.TYPE_FATAL, new Object[] {"Object Code", originEntry.getChartOfAccountsCode() + "-" + originEntry.getFinancialObjectCode()});
         }
         if(ObjectUtils.isNull(account.getSubFundGroup())){
-            throw new IllegalArgumentException("This account specified "+originEntry.getChartOfAccountsCode()+"-"+originEntry.getAccountNumber()+" does not exist. For sequence "+originEntry.getTransactionLedgerEntrySequenceNumber());
+            return MessageBuilder.buildMessageWithPlaceHolder(edu.arizona.kfs.sys.KFSKeyConstants.ERROR_GLOBAL_TRANSACTION_EDIT_SCRUBBER_INVALID_VALUES, Message.TYPE_FATAL, new Object[] {"Sub Fund", account.getSubFundGroupCode()});
```

Comment:
Aesthetics: Do we need to type the entire package for the error key: edu.arizona.kfs.sys.KFSKeyConstants.ERROR_GLOBAL_TRANSACTION_EDIT_SCRUBBER_INVALID_VALUES

Maybe import the package?